packages/odf/components/storage-consumers/onboarding-modal.tsx

```diff
@@ -115,9 +171,27 @@ export const ClientOnBoardingModal: ClientOnBoardingModalProps = ({
           <ModalBody>
             <p className="odf-onboarding-modal__title-desc">
               {t(
-                'Add storage capacity for the client cluster to consume from the provider cluster.'
+                `An onboarding token to authenticate and authorize an OpenShift cluster, granting access to the Data Foundation deployment, thus establishing a secure connection.`
               )}
             </p>
+            <Flex direction={{ default: 'column' }}>
+              <FlexItem>
+                <span className="odf-onboarding-modal__quota-desc">
+                  <Title headingLevel="h5" size="md">
+                    {t('StorageConsumer')}:{' '}
```

Comment:
Is the code formatter adding it?
```suggestion
                    {t('StorageConsumer')}:
```